controllers/machineset_controller.go

```diff
@@ -59,6 +59,11 @@ var (
 	stateConfirmationInterval = 100 * time.Millisecond
 )
 
+//go:generate counterfeiter . machineHealthCheck
+type machineHealthCheck interface {
```

Comment:
This isn't actually healthy checking, perhaps it should be `machineRemediator` instead?